fix(plugin-whatsapp): remove forbidden TypeScript import suffix

## Problem

Merged PR #22558 added `./whatsapp-target-prefix.ts` to `plugins/plugin-whatsapp/src/normalize.ts`. The package does not enable `allowImportingTsExtensions`, so its required typecheck fails with TS5097.

## Reproduction

```text
bun run --cwd plugins/plugin-whatsapp typecheck
src/normalize.ts(9,45): error TS5097: An import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.
```

## Acceptance

- Import uses the repository's ESM-compatible extensionless source convention.
- WhatsApp typecheck, lint, build, and focused tests pass.
- Root `bun run verify` advances past the WhatsApp lane.

AI provider/model: OpenAI / gpt-5.6-sol
Client / agent tooling: Codex desktop
Contribution skill revision: elizaOS/eliza@bad793372ea9a08fd91c97c8cd9dfc34fc84a24c:packages/skills/skills/contribute-to-eliza
Attribution status: self-reported
— [codex-computer-use]
<!-- eliza-computer-attribution:v1 {"provider":"openai","model":"gpt-5.6-sol","client":"Codex desktop","skill_revision":"elizaOS/eliza@bad793372ea9a08fd91c97c8cd9dfc34fc84a24c:packages/skills/skills/contribute-to-eliza"} -->

--- a/plugins/plugin-whatsapp/src/normalize.ts
+++ b/plugins/plugin-whatsapp/src/normalize.ts
@@ -6,7 +6,7 @@
  */
 
 import { truncateWellFormed } from "@elizaos/core";
-import { stripWhatsAppTargetPrefixes } from "./whatsapp-target-prefix.ts";
+import { stripWhatsAppTargetPrefixes } from "./whatsapp-target-prefix";
 
 /**
  * WhatsApp text chunk limit

